db/migrate/20200917144140_create_administrators.rb

```diff
@@ -0,0 +1,8 @@
+class CreateAdministrators < ActiveRecord::Migration[6.0]
+  def change
+    create_table :administrators do |t|
+      t.belongs_to :member, foreign_key: true
```

Comment:
This is where we can see a foreign key being brought in as a column. The column here is named `member_id` because `member` is the foreign reference, but we *may* be able to rename this by providing a name parameter.